### DISCUSSION TITLE
filtering of Cyrillic characters [Sep 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6358] [Discord Threads]
> 🙋‍♂️ **asukhan**

**Comment by o1lab** [comment]
> 🙋‍♂️ **asukhan**  
#### Can you please tell me if there are plans to improve the filtering of Cyrillic characters?  
#  
> 💁 **o1lab**  
#### Can u pls help raise an issue for this in github  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1135871016845967420)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
